Source/Core/RequestScheduler.js

```diff
@@ -57,15 +170,34 @@ define([
     };
 
     /**
+     * Get the request server from a given url.
+     *
+     * @param {String} url The url.
+     * @returns {RequestServer} The request server.
+     */
+    RequestScheduler.getRequestServer = function(url) {
+        var requestServer;
+        var serverName = RequestScheduler.getServerName(url);
+        var length = activeRequestsByServer.length;
```

Comment:
Why is `activeRequestsByServer` an array now?  Wouldn't it be better as an object used as a hashmap?